app/src/main/java/org/mifos/selfserviceapp/ui/fragments/HomeFragment.java

```diff
@@ -86,8 +90,18 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         ((HomeActivity) getActivity()).getActivityComponent().inject(this);
 
         ButterKnife.bind(this, rootView);
-
         presenter.attachView(this);
+
+        slHomeContainer.setColorSchemeResources(R.color.blue_light, R.color.green_light, R
+                .color.orange_light, R.color.red_light);
+        slHomeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
```

Comment:
@dilpreet96 do this `setOnRefreshListener(this)` and implement `onRefresh` and like wise you are using same code twice so better make method for them and use that 
for this 
` 
presenter.loadClientAccountDetails();
presenter.getUserDetails();
presenter.getUserImage();
`